### PR TITLE
Fix wrap in base64-ed data

### PR DESCRIPTION
**Pull request by whtsky**
`base64.encodebytes` would insert newlines (`\n`) asfter every 76 bytes of output, thus produce invalid data url.

https://docs.python.org/3.8/library/base64.html#base64.encodebytes

### CODE REVIEW

**Comment by whtsky** [comment]
https://travis-ci.org/github/fcurella/python-datauri/jobs/713212702

CI failed because of network failure, but I don't have permission to retry it.

**Comment by fcurella** [comment]
> should I just remove it from travis matrix?

Yes please :)

**Comment by fcurella** [comment]
@whtsky while you're at it, would you also remove python 2.7?

**Comment by whtsky** [comment]
> @whtsky while you're at it, would you also remove python 2.7?

Travis still supports Python 2.7. Do you want to completely remove support for Python 2?

File: tests/test_long_file.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Four score and seven years ago our fathers brought forth on this continent, a new nation, conceived in Liberty, and dedicated to the proposition that all men are created equal.
```
[the remark this answers is not in the document]
Comment (whtsky) [reply tests/test_long_file.txt]: changed to Lorem Ipsum.

**Comment by fcurella** [comment]
On a second thought, we can remove python 2.7 in a separate PR. I'd like to remove all the shims that will not be necessary anymore.

**Comment by fcurella** [comment]
Thank you!

**Pull request merged**